src/components/SkipButton/SkipButton.test.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
```

Comment:
We have discussed before (CI) to keep tests out of src folder in their specs folded.

What do you think about that? Would like to know your opinion @v-i-s-h-n-u-ps

Reply:
I don't have strong preferences. @esezen what do you think?

Reply:
I saw this after mentioning it in my review. I kinda do have a strong preference just because we follow that pattern in other libraries and I would love to keep it consistent if possible

Reply:
Side note about this, since tests files are under `src` our compile step is trying to build them and it's producing some errors. Depending on where we land with this decision, we might need to look into that